runtime/lua/vim/lsp/client.lua

```diff
@@ -113,10 +112,11 @@ local validate = vim.validate
 --- Sends a request to the server.
 --- This is a thin wrapper around {client.rpc.request} with some additional
 --- checking.
---- If {handler} is not specified,  If one is not found there, then an error
---- will occur. Returns: {status}, {[client_id]}. {status} is a boolean
---- indicating if the notification was successful. If it is `false`, then it
---- will always be `false` (the client has shutdown).
+--- If {handler} is not specified and if there's no respective server-side
+--- handler, then an error will occur.
+--- Returns: {status}, {[client_id]}. {status} is a boolean indicating if
+--- the notification was successful. If it is `false`, then it will always
+--- be `false` (the client has shutdown).
```

Comment:
This is the main change in this PR. @lewis6991 I found this description confusing, lmk if I got it right.

Reply:
> if there's no respective server-side handler

This doesn't look correct to me?

The code is:
```lua
  if not handler then
    handler = assert(
      self:_resolve_handler(method),
      string.format('not found: %q request handler for client %q.', method, self.name)
    )
  end
```

and `_resolve_handler` is just:
```lua
function Client:_resolve_handler(method)
  return self.handlers[method] or lsp.handlers[method]
end
```

So I'd replace "server-side" handler with "global" or "default" handler.